portal-api/src/v1/controllers/reports/unissued-facilities.controller.js

```diff
@@ -107,7 +107,7 @@ exports.findUnissuedFacilitiesReports = async (req, res) => {
         facility.daysLeftToIssue = defaultDate ? differenceInCalendarDays(todaysDate, deadlineForIssuing) * -1 + 0 : 0;
 
         const defaultFacilityValue = item.value ? parseInt(item.value, 10) : 0;
-        facility.currencyAndValue = item.value ? `${item.currency} ${formattedNumber(defaultFacilityValue)}` : '';
+        facility.currencyAndValue = item.value ? `${item.currency.id} ${formattedNumber(defaultFacilityValue)}` : '';
```

Comment:
the `id` shouldn't be here. in the controller I'm already getting the currency. have a look at this
https://github.com/UK-Export-Finance/dtfs2/blob/allFacilities/portal-api/src/v1/controllers/reports/unissued-facilities.controller.js#L48

Reply:
Right you are, fixed